en-US/Design.xml

```diff
@@ -371,6 +372,89 @@ $ vi myFile.txt
+      </para>
+      <note><title>Classroom Exceptions</title>
+        <para>
+          Although security is important, it is more important to not have unnecessary classroom security distract from the immediate topic being taught.
+        </para>
+      </note>
+      <section><title>General Recommendations</title>
+        <note>
+          <para>
+            These are recommendations, not rules.
+            As with most things, consistency is important.
+            Do not swap between different approaches without reason.
+            Choose which approach works best for your situation and use it consistently.
+          </para>
+        </note>
+        <itemizedlist>
+          <listitem>
+            <para>
+              In all cases, use the minimum privilege level required to achieve the task.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              In exercises, use <command>sudo</command> and <command>sudo -i</command> and set this up to work throughout all relevant systems in the classroom.
+              Do not use <command>su -</command> without good cause.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              When there is a scattered minority of privileged commands in a mostly unprivileged exercise, use <command>sudo</command> on a per-command basis.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              When the exercise is majority privileged, or has a significant number of privileged commands, use <command>sudo -i</command> either at the beginning of the exercise, or at an appropriate step where the privileged commands begin.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              In the narrative, do not show the use of <command>su</command> or <command>sudo</command>, but always show privileged commands with the correct prompt.
+              See <xref linkend="exam-documenting-command-syntax"/> for information about command prompts.
+            </para>
+          </listitem>
+        </itemizedlist>
+      </section>
+      <section><title>Exceptions</title>
+        <para>
+          Some courses are specifically designed to teach <command>sudo</command> and its variations, the use of the related files, such as <filename>/etc/sudoers</filename> and so on.
+          For these courses, use the required variation for the topic being taught.
+        </para>
+        <section><title>Ansible Courses</title>
+          <itemizedlist>
+            <listitem>
+              <para>
+                Ansible courses typically use a <systemitem>devops</systemitem> user with passwordless <command>sudo ALL=ALL(ALL)</command> access on managed nodes to enable the use of <command>become</command> without a <command>become</command> password as <systemitem>root</systemitem> to do anything.
```

Comment:
I think `command` should be replaced with `code` tags, and include `NOPASSWD:` if it needs to be passwordless.

Reply:
I can change the tags, no problem, but perhaps you could do a `Ctrl+G` and make a suggestion about the NOPASSWD: bit to make sure I get it right? thanks

Reply:
```suggestion
                Ansible courses typically use a <systemitem>devops</systemitem> user with passwordless sudo access (<code>devops	ALL=(ALL)	NOPASSWD: ALL</code>) on managed nodes to enable the use of <command>become</command> without a <command>become</command> password as <systemitem>root</systemitem> to do anything.
```